joern-cli/frontends/c2cpg/src/main/scala/io/joern/c2cpg/astcreation/AstCreator.scala

```diff
@@ -40,14 +36,13 @@ class AstCreator(
   // To achieve this we need this extra stack.
   protected val methodAstParentStack: Stack[NewNode] = new Stack()
 
-  def createAst(): Unit =
+  def createAst(): DiffGraphBuilder = {
     Ast.storeInDiffGraph(astForFile(parserResult), diffGraph)
+    diffGraph
+  }
 
   private def astForFile(parserResult: IASTTranslationUnit): Ast = {
-    val cpgFile            = Ast(NewFile().name(filename).order(0))
```

Comment:
Is the `NewFile` gone now by accident?

Reply:
No, I removed it because the backend passes create these file nodes automatically when they encounter `filename` fields.